index.html

```diff
@@ -208,16 +208,27 @@ <h5>Get started</h5>
     </div>
 
     <ul class="form-name-box">
-      <li class="form-name-gray">Your name</li>
-      <li class="fill-name">Anastasiia</li>
+      <li class="form-name-gray"><label for="name">Your name</label></li>
+      <li class="fill-name">
+        <input type="text" name="name" id="name" value="Anastasiia" placeholder="Your name" maxlength="30" required>
+        <input>
+      </li>
+
     </ul>
 
     <ul class="form-email-box">
-      <li class="form-email-gray">Your email</li>
-      <li class="fill-email">|</li>
+      <li class="form-email-gray"><label for="text">e-mail</label></li>
+      <li class="fill-email"><input type="text" name="e-mail" id="text" value="" placeholder="|" maxlength="30"
```

Comment:
- [ ] You should use `type="email"` because this field is email and doing so helps for validation and others.

Reply:
<form action="https://formspree.io/f/myyobvpo" method="Post" class="form-name-box">
      <div class="form-name-gray"><label for="name">Your name</label></div>
      <div class="fill-name">
        <input type="text" name="name" id="name" value="" placeholder="Your name" maxlength="30" required>
      </div>

    </form>

    <form action="https://formspree.io/f/myyobvpo" method="POST" class="form-email-box">
      <div class="form-email-gray"><label for="text">e-mail</label></div>
      <div class="fill-email"><input type="text" name="e-mail" id="text" value="" placeholder="|" maxlength="30"
          required>
      </div>
    </form>

    <form class="message-box">
      <div class="fill-messagge"><label for="text"></label>
        <textarea name="name" id="form" cols="32" rows="7" maxlength="500" wrap="soft"
          placeholder="Enter your message here..." required></textarea>

      </div>
    </form>

Reply:
Good job in implementing the changes 👍